Core/Core/Extensions/UIScrollViewExtensions.swift

```diff
@@ -15,15 +15,9 @@
 //
 
 import Foundation
-import UIKit
-import Core
 
-class ModuleListCell: UITableViewCell {
-    @IBOutlet weak var nameLabel: UILabel!
-
-    var module: Module? {
-        didSet {
-            nameLabel.text = module?.name
-        }
+extension UIScrollView {
+    public var isEndReached: Bool {
+        return contentOffset.y >= (contentSize.height - frame.size.height)
```

Comment:
do we only ever care about vertical scrolling? I think we can name this to indicate `Bottom` instead of `End`, or maybe check the scroll direction?

Also, if we are using this for paging, would we want a threshold rather than making them get all the way to very bottom before we load more?

Reply:
Both good ideas. I'll name it `Bottom` and add a threshold of... 50? That's about the size of a row.. Or maybe 80? More like two rows?

Reply:
I made it a function with optional `threshold` input with a default of 60 (so about a row and a half).